derp/derphttp/derphttp_client.go

```diff
@@ -142,6 +149,15 @@ func NewClient(privateKey key.NodePrivate, serverURL string, logf logger.Logf) (
 	return c, nil
 }
 
+// Started reports whether this client has been used yet.
+//
+// If if reports false, it may still have its exported fields configured.
+func (c *Client) Started() bool {
```

Comment:
does this need to be exported?

Reply:
I guess not, huh? I thought it was going to need to be, but I'll unexported it for now.